right_arm_dofs:
  - rShoulderY1
  - rShoulderX
  - rShoulderY2
  - rElbowZ
  - rElbowX
  - rElbowY
  - rWristZ
  - rWristX
  - rWristY

left_arm_dofs:
  - lShoulderY1
  - lShoulderX
  - lShoulderY2
  - lElbowZ
  - lElbowX
  - lElbowY
  - lWristZ
  - lWristX
  - lWristY

right_arm_default:
  rShoulderX: 0.
  lShoulderX: 0.
  rShoulderTransX: .018
  rShoulderTransY: .33
  rShoulderTransZ: .25
  rArmTrans: .39
  rForeArmTrans: .24
